smart-contracts/wasm-chain-integration/src/constants.rs

```diff
@@ -4,15 +4,12 @@ pub const MAX_CONTRACT_STATE: u32 = 16384; // 16kB
 /// Maximum number of nested function calls.
 pub const MAX_ACTIVATION_FRAMES: u32 = 1024;
 
-/// Maximum size of the init/receive parameter.
-pub const MAX_PARAMETER_SIZE: usize = 1024;
-
 /// Maximum size of a log message.
 pub const MAX_LOG_SIZE: u32 = 512;
 
-/// Maximum number of log messages per execution.
-/// This together with the previous constant limits the amount of data that can
-/// be logged to 16kB.
+/// Maximum number of log messages per execution *in protocol version < 5* (no
+/// limit from 5 and onward). This together with the previous constant limits
+/// the amount of data that can be logged to 16kB.
```

Comment:
Isn´t it bound by the block now (p5 and onwards) as opposed to the 16kb specified here?

Reply:
Yes, I tried to make that clear, but apparently, it wasn't :p. I've rephrased it now ;)